app/components/modals/simple-modal.js

```diff
@@ -9,4 +9,10 @@ export default class SimpleModal extends ModalBase {
     this.close();
     this.action();
   }
+
+  @action
+  performCancel() {
+    this.close();
+    this.router.transitionTo('public', { queryParams: { video_dialog: null } });
```

Comment:
Seriously? This component is not just there for video dialog, it is a generic component